src/Native/Unix/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -99,22 +109,6 @@ else()
       ${OPENSSL_CRYPTO_LIBRARY}
       ${OPENSSL_SSL_LIBRARY}
     )
-  
-    # On OS X every library emits the manner in which it should be referenced.
-    # All of our libraries are referenced via @rpath, which is similar to how Linux and Windows
-    # libraries are loaded. The homebrew installation of OpenSSL (libcrypto, libssl) uses the
-    # full path to the library installation. This means that this library is not flexible to
-    # users installing newer libcrypto in the working directory, or to systems which do not
-    # install to the same path as homebrew does.
-    #
-    # So, after compiling, rewrite the references to libcrypto to be more flexible.
-    if (APPLE)
```

Comment:
It is possible that this change would cause troubles to some people?

Reply:
This was already wrong if it got built against OpenSSL 1.1 (which may be part of the problem in the dotnet/runtime repo), since it didn't rewrite-to-rpath the 1.1 name.

Part of why went with "only/always" was so that we can avoid this sort of hackery (only having the "portable loader" sort of hackery :smile:).

Reply:
Ok, makes sense.